lib/pengine/pe_output.c

```diff
@@ -1143,9 +1145,7 @@ cluster_options_xml(pcmk__output_t *out, va_list args) {
     }
 
     pcmk__output_create_xml_node(out, PCMK_XE_CLUSTER_OPTIONS,
-                                 PCMK_OPT_STONITH_ENABLED,
-                                 pcmk__flag_text(scheduler->flags,
-                                                 pcmk_sched_fencing_enabled),
+                                 PCMK_XA_STONITH_ENABLED, stonith_enabled,
```

Comment:
I'm on the fence about this series of changes. All the XML attributes in this function call are the names of cluster options. Previously they began with `PCMK_OPT`. This series of commits adds `PCMK_XA` equivalents to use in the XML attribute context. (`PCMK_OPT` is used as the name of an nvpair.)

I could be convinced to leave these as just `PCMK_OPT`, even here.